src/tests/system/tests/test_ipa.py

```diff
@@ -122,3 +124,70 @@ def test_ipa__hostpublickeys_by_ip(client: Client, ipa: IPA, public_keys: list[s
     assert len(public_keys) == len(result.stdout_lines), "Did not get expected number of public keys!"
     for key in public_keys:
         assert f"{ip} {key}" in result.stdout_lines, "Did not get expected public keys!"
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__check_gssapi_authentication_indicator(client: Client, ipa: IPA):
+    """
+    :title: Verify GSSAPI authentication indicator
+    :description:
+        Checks that the kerberos ticket was pre-authentication using GSSAPI using authentication indicators
+    :setup:
+        1. Configure SSSD for sudo and gssapi
+        2. Start SSSD
+        3. Create sudo configuration that allows user to run SUDO rules
+    :steps:
+        1. Switch to ipa user
+        2. Obtain Kerberos ticket.
```

Comment:
Collapse step 1 and 2 please.

1. Login as the test user and obtain ticket

:expectedresult:

1. Login successful and ticket obtained